### PR TITLE
AndrewBalint Branch

### PR DESCRIPTION
**Pull request by andrewbalint51**
Structured as 3 methods in one utility class.

### CODE REVIEW

File: src/AndrewBalint/TaskUtilityMethods.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        System.out.println(identify(11));
+
+        System.out.println(NDivide(99,25));
+
+        FINRA();
+
+    }
+
+    public static String identify(int num){
```
Comment (AndrewSloth) [inline src/AndrewBalint/TaskUtilityMethods.java:16]: Greate work.
But I belive your method "identify" should work with all numbers. For now it working only with 0 and positive numbers.  
Correct me if I wrong.

File: src/AndrewBalint/TaskUtilityMethods.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    public static int NDivide(int n1, int n2){
+        int result =0;
+
+        while(n1>=n2){
+            n1-=n2;
+            result++;
+        }
+
```
Comment (AndrewSloth) [inline src/AndrewBalint/TaskUtilityMethods.java:45]: And I also belive method "NDivide" should also work with all numbers. It don't work if you tried divide -30 / 10

**Review by AndrewSloth — CHANGES_REQUESTED** [review]
Greate work.
But I belive your method "identify" should work with all numbers. For now it working only with 0 and positive numbers.
And I also belive method "NDivide" should also work with all numbers. It won't work if you tried divide -30 / 10
Correct me if I wrong.

**Comment by andrewbalint51** [comment]
No you're right good catch. Fixed.

**Review by AndrewSloth — APPROVED** [review]

**Pull request merged**